client/src/services/api.tsx

```diff
@@ -16,7 +16,10 @@ const api = axios.create({
 api.interceptors.response.use(
   response => response,
   error => {
-    if (error?.response?.data?.message && error.response.status === 400) {
+    if (
```

Comment:
Maybe rather than checking for ranges of codes such as 
`error.response.status >= 400 && error.response.status <= 599` or something along the lines of that, in order to catch all errors?